src/ui/PrintableUri/PrintableUri.ts

```diff
@@ -12,7 +12,9 @@ export interface IPrintableUriOptions {
 }
```

Comment:
I prefer to document methods on components that have significant code coverage, so we can ensure their backward compatibility.

This particular component is unfortunately poorly tested, and we need coverage before documenting them (and making them official)